src/auth.js

```diff
@@ -362,18 +362,22 @@ export default class DropboxAuth {
 
     const headers = {};
     headers['Content-Type'] = 'application/json';
-    refreshUrl += `?grant_type=refresh_token&refresh_token=${this.getRefreshToken()}`;
-    refreshUrl += `&client_id=${clientId}`;
+    const fetchOptions = {
+      method: 'POST',
+    };
+
+    const body = { grant_type: 'refresh_token', client_id: clientId };
+
     if (clientSecret) {
-      refreshUrl += `&client_secret=${clientSecret}`;
+      body.client_secret = clientSecret;
+      body.refresh_token = this.getRefreshToken();
     }
+
     if (scope) {
-      refreshUrl += `&scope=${scope.join(' ')}`;
+      body.scope = scope.join(' ');
     }
-    const fetchOptions = {
-      method: 'POST',
-    };
 
+    fetchOptions.body = body;
```

Comment:
I've tried not to change the original style, but we could also write it like this:

```js
    const fetchOptions = {
      body,
      headers
      method: 'POST',
    };
```